frontend/src/components/header/Header.tsx

```diff
@@ -51,98 +26,107 @@ export default function Header({ isAuthenticated }: { isAuthenticated: boolean }
                         <span className="sr-only">Burgieclan</span>
                         <Logo width={50} height={50}/>
                     </a>
-                    <div className="flex">
-                        <Input ref={searchInputRef} id="search" name="search" type="search" placeholder="search"/>
-                    </div>
+                    {isAuthenticated && <Search />}
                 </div>
 
                 {/* Mobile menu toggle button */}
-                <div className="flex md:hidden">
-                    <button
-                        type="button"
-                        onClick={() => setMobileMenuOpen(true)}
-                        className="-m-1.5 p-1.5 w-[50px] h-[50px] inline-flex items-center justify-center rounded-md text-gray-700"
-                    >
-                        <span className="sr-only">Open main menu</span>
-                        <Bars3Icon aria-hidden="true" className="h-6 w-6"/>
-                    </button>
-                </div>
+                <Suspense fallback={<Skeleton width={100} height={20}/>}>
```

Comment:
Type error: Type '{ width: number; height: number; }' is not assignable to type 'IntrinsicAttributes & HTMLAttributes<HTMLDivElement>'.
  Property 'width' does not exist on type 'IntrinsicAttributes & HTMLAttributes<HTMLDivElement>'.
  
  Copilot